lighthouse-core/audits/byte-efficiency/unused-css-rules.js

```diff
@@ -66,14 +63,44 @@ class UnusedCSSRules extends ByteEfficiencyAudit {
       }
 
       if (rule.used) {
-        stylesheetInfo.used.push(rule);
-      } else {
-        unused++;
-        stylesheetInfo.unused.push(rule);
+        stylesheetInfo.usedRules.push(rule);
       }
     });
+  }
+
+  /**
+   * @param {!Object} stylesheetInfo
+   * @param {boolean} isInline
+   * @return {{debugString: string|undefined, wastedBytes: number, totalBytes: number}}
+   */
+  static computeUsage(stylesheetInfo, isInline) {
+    let usedUncompressedBytes = 0;
+    const totalUncompressedBytes = stylesheetInfo.content.length;
+
+    for (const usedRule of stylesheetInfo.usedRules) {
+      usedUncompressedBytes += usedRule.endOffset - usedRule.startOffset;
+    }
+
+    // If we don't know for sure how many bytes this sheet used on the network,
+    // we can guess it was roughly the size of the content gzipped.
+    const totalBytes = stylesheetInfo.networkRecord ?
+        stylesheetInfo.networkRecord.transferSize :
+        Math.round(stylesheetInfo.content.length / 3);
```

Comment:
since this is the full run, not the default run, instead of estimating like this can we just actually run gzip on it like `response-compression` does?